include/gridtools/stencil/cpu_ifirst/entry_point.hpp

```diff
@@ -31,6 +31,9 @@
 namespace gridtools {
     namespace stencil {
         namespace cpu_ifirst_backend {
+            template <class PlhInfo>
+            using get_extent_f = typename PlhInfo::extent_t;
```

Comment:
You do not need to define this alias here. This is already at your service as `be_api::get_extent`